### PR TITLE
changes based on seed step onboarding

### PR DESCRIPTION
**Pull request by kpslp**

### CODE REVIEW

File: docs/register-intune-device-id.md [hunk 1]
```diff
@@ -19,25 +19,44 @@ Following are the prerequisites to register Intune device ID on TechPass portal:
...
+
+4. Ensure the device you are onboarding is connected to the Internet so that Intune is able to install the required software and configurations.
+
+5. Refer to the following table to know about the possible onboarding status and the action required by you.
+
+| Status | Description | Action required |
+|---| ---| ---|
+| **triggered, waiting for software installation (step 1 of 2)**| Your SEED onboarding has been triggered on the device and is waiting for the software installation to be completed. | When you click the refresh button after a successful software installation, the status changes to **software installed, waiting for backend onboarding**.|
+| **software installed, waiting for backend onboarding (step 2 of 2)**| Required software has been installed on the device and waiting for backend onboarding.  | When you click the refresh button after a successful backend onboarding, the status changes to **onboarded** . |
+| **onboarded** | Your SEED onboarding is successful. | Go to step 8 in this section.  |
```
Comment (JLeeChan) [inline docs/register-intune-device-id.md]: This should be step 6.

File: docs/register-intune-device-id.md [hunk 2]
```diff
@@ -19,25 +19,44 @@ Following are the prerequisites to register Intune device ID on TechPass portal:
...
\ No newline at end of file
+  ![enter-intune](assets/images/intune-id/macos-device-listed-tp-portal.png)
+
+4. Ensure the device you are onboarding is connected to the Internet so that Intune is able to install the required software and configurations.
+
+5. Refer to the following table to know about the possible onboarding status and the action required by you.
+
+| Status | Description | Action required |
+|---| ---| ---|
+| **triggered, waiting for software installation (step 1 of 2)**| Your SEED onboarding has been triggered on the device and is waiting for the software installation to be completed. | When you click the refresh button after a successful software installation, the status changes to **software installed, waiting for backend onboarding**.|
```
Comment (JLeeChan) [inline docs/register-intune-device-id.md]: The complete next status should have **(step 2 of 2)**

File: docs/register-intune-device-id.md [hunk 3]
```diff
@@ -19,25 +19,44 @@ Following are the prerequisites to register Intune device ID on TechPass portal:
```
Comment (JLeeChan) [inline docs/register-intune-device-id.md:19]: I couldn't add a comment so capturing it here. A few lines up there is a  service request link pointing to https://go.gov.sg/techpass-sr 
image

File: docs/register-intune-device-id.md (hunk 2 above)
Comment (kpslp) [reply docs/register-intune-device-id.md]: sorry, i dont quite get it? i think step 2 of 2 is in software installed etc. this, i took straight from the seed docs

File: docs/register-intune-device-id.md (hunk 3 above)
Comment (kpslp) [reply docs/register-intune-device-id.md:19]: is that ok or...?

File: docs/register-intune-device-id.md (hunk 2 above)
Comment (JLeeChan) [reply docs/register-intune-device-id.md]: The **software installed, waiting for backend onboarding** should be **software installed, waiting for backend onboarding (step 2 of 2)**

File: docs/register-intune-device-id.md (hunk 1 above)
Comment (kpslp) [reply docs/register-intune-device-id.md]: fixed

File: docs/register-intune-device-id.md (hunk 2 above)
Comment (kpslp) [reply docs/register-intune-device-id.md]: done

**Review by JLeeChan — APPROVED** [review]

**Pull request merged**